addons/common/fnc_removePerFrameHandler.sqf

```diff
@@ -25,8 +25,12 @@ Author:
 #include "script_component.hpp"
 
 params ["_publicHandle"];
+
+if (isNil "_publicHandle" || {(_publicHandle < 0)} || {!((count GVAR(PFHhandles)) >= _publicHandle)}) exitWith { // Nil/no handle, nil action or handle is larger than Public Handle Array
```

Comment:
Should this be `{(count GVAR(PFHhandles) <= _publicHandle)}`?

If the count is 0, and the _publicHandle is 0, you want it to be true and exit.